tests/benchmarks/latency_measure/testcase.yaml

```diff
@@ -54,3 +54,25 @@ tests:
         regex: "(?P<metric>.*) - (?P<description>.*):(?P<cycles>.*) cycles ,(?P<nanoseconds>.*) ns"
       regex:
         - "PROJECT EXECUTION SUCCESSFUL"
+
+  benchmark.kernel.latency.objcore:
+    # FIXME: no DWT and no RTC_TIMER for qemu_cortex_m0
+    platform_exclude:
+      - qemu_cortex_m0
+      - m2gl025_miv
+    filter: CONFIG_PRINTK and not CONFIG_SOC_FAMILY_STM32
+    timeout: 300
+    extra_configs:
+      - CONFIG_OBJ_CORE=y
+      - CONFIG_OBJ_CORE_STATS=y
```

Comment:
This additional test suite allows to estimate extra latency the obj_core & stats feature introduces (in comparison to the `benchmark.kernel.latency` test suite results).